internal/docstore/dynamodocstore/codec.go

```diff
@@ -320,6 +331,12 @@ func toGoValue(av *dyn.AttributeValue) (interface{}, error) {
 	}
 }
 
-func (decoder) AsSpecial(reflect.Value) (bool, interface{}, error) {
-	return false, nil, nil
+func (d decoder) AsSpecial(v reflect.Value) (bool, interface{}, error) {
+	switch v.Type() {
+	case typeOfGoTime:
+		t, err := time.Parse(time.RFC3339Nano, *d.av.S)
```

Comment:
You have to check that the `S` field is populated. Return an error if it isn't.

Reply:
Done.